poc/gateway-authpolicy/test.sh

```
@@ -0,0 +1,178 @@
+GATEWAY_HOST="${GATEWAY_HOST:-maas.apps.your-cluster.example.com}"
+MODEL_NS="${MODEL_NS:-llm}"
+MODEL_NAME="${MODEL_NAME:-granite-3b}"
+VALID_KEY="${VALID_KEY:-sk-oai-REPLACE_ME}"
+INVALID_KEY="${INVALID_KEY:-sk-oai-invalid}"
+
+# Path-based inference URL (current MaaS routing)
+PATH_URL="https://${GATEWAY_HOST}/${MODEL_NS}/${MODEL_NAME}/v1/chat/completions"
+# Body-based inference URL (BBR / Approach C)
+BODY_URL="https://${GATEWAY_HOST}/v1/chat/completions"
+# Models listing (should bypass auth check)
+MODELS_URL="https://${GATEWAY_HOST}/v1/models"
```

Shellcheck (0.11.0)

[warning] MODELS_URL appears unused. Verify use (or export if used externally).

(SC2034)